service/migration_manager.cc

```diff
@@ -103,19 +103,20 @@ void migration_manager::init_messaging_service()
             return db::schema_tables::recalculate_schema_version(_sys_ks, _storage_proxy.container(), _feat);
         });
     };
+    auto reload_schema_in_bg = [this] {
```

Comment:
Maybe we should just get rid of `update_schema` and use `reload_schema_in_bg` everywhere? IIUC, `reload_schema_in_bg` does everything that `update_schema` does and then some.

This should make it harder to make this mistake again in the future.

@tgrabiec WDYT?

Reply:
Makes sense.